pkg/poolcache/poolcache.go

```diff
@@ -52,13 +56,15 @@ type (
 		function        interface{}
 		address         interface{}
 		value           interface{}
+		requestsPerPod  int
+		cpuUsage        resource.Quantity
```

Comment:
`cpuUsage` is actual usage or a place to store limit for that function?

Reply:
Its used for updating both CPU usage and CPU limit